tsdb/head.go

```diff
@@ -1527,6 +1527,7 @@ type memSeries struct {
 	txs *txRing
 
 	// Temporary variable for sparsehistogram experiment.
+	// true if the head chunk in the series is for histograms.
```

Comment:
true → True

Perhaps it's better to rename the variable to be more descriptive so that we don't even need the comment? (Also, the old comment ("Temporary variable for sparsehistogram experiment.") is kind of weird. The same could be said about _all_ the sparsehistogram code.)

Reply:
Changing it to `isHistogramSeries`